### PR TITLE
Update documentation to use INSTALL step under Windows builds

### PR DESCRIPTION
**Pull request by Foereaper**
### Description

Updated documentation to provide clearer and easier installation instructions, using the built-in INSTALL project provided by cmake.

Using the install step is best practice, and you should never really run your production server out of your build folder. The build folder should *always* be considered volatile.

### Related Issue

Closes

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

### CODE REVIEW

**Comment by Helias** [comment]
thanks for your PR, there is a merge conflicts, please resolve it,if you need any help comment below

**Comment by Kitzunu** [comment]
I dont like this for several reasons.
It makes it messier and unnecessary complicated to start up. And there is nothing wrong with starting it from the build folder.

File: docs/windows-core-installation.md [hunk 1]
```diff
@@ -151,7 +169,9 @@ In the **configs** folder you should find:
 
 ```
 authserver.conf.dist
+authserver.conf
 worldserver.conf.dist
+worldserver.conf
```
Comment (Kitzunu) [inline docs/windows-core-installation.md:174]: Wrong. The .conf files arent created at build

Comment (Kitzunu) [reply docs/windows-core-installation.md:174]: Well more like not needed to run the server. It comes later in the guide. Adding it here could just cause confusion

File: docs/server-setup.md [hunk 2]
```diff
@@ -78,11 +78,7 @@ You need to run Mapextractor.exe before the makevmaps_simple.bat.
 
 ### Creating the config files
 
-1. Make copies of both .dist files.
-
-1. From each copy, remove the .dist part.
```
Comment (Kitzunu) [inline docs/server-setup.md:83]: Removing this is wrong

File: docs/server-setup.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Github link:
 
 1. Download the files above.
 
-1. Create a new folder within the build folder called **Data**. i.e windows: **C:\Build\bin\RelWithDebInfo\Data** or linux: **$HOME/azeroth-server/data/**
+1. Create a new folder within the server folder you created in the previous step, in this case **C:\Server**, called **Data**. i.e **C:\Server\Data**. For linux: **$HOME/azeroth-server/data/**
```
Comment (Kitzunu) [inline docs/server-setup.md:30]: You just make it wordy for no reason

File: docs/windows-core-installation.md (hunk 1 above)
Comment (Foereaper) [reply docs/windows-core-installation.md:174]: No, with the install step it automatically creates the conf files and not just the dist files, that's part of how your install script works in cmake 😄

File: docs/server-setup.md (hunk 2 above)
Comment (Foereaper) [reply docs/server-setup.md:83]: See above comment :)

File: docs/server-setup.md (hunk 3 above)
Comment (Foereaper) [reply docs/server-setup.md:30]: Since it's a separate page, it makes sense to reference back to the previous step, or at least I think so.

**Pull request closed without merging**